org.tenacityaudio.Tenacity.yaml

```diff
@@ -176,6 +176,22 @@ modules:
         commit: 147dd722548358763a8b649b3e4b41dfffbcfbb6
     buildsystem: cmake-ninja
 
+  - name: portsmf
+    sources:
+      - type: git
+        url: https://github.com/tenacityteam/portsmf.git
+        tag: '236'
+        commit: 896b8fc8ed440c8f2eb40e518229897431529b4e
+    buildsystem: cmake-ninja
+
+  - name: portmidi-mixxxdj
+    sources:
+      - type: git
+        url: https://github.com/mixxxdj/portmidi.git
+        #tag: '236'
```

Comment:
Why is this commented out?

Reply:
I was waiting for you to release it.

Reply:
I had already tagged it but I just made a GitHub release out of the tag: https://github.com/mixxxdj/portmidi/releases/tag/236